content/learn/book/getting-started/setup/_index.md

```diff
@@ -105,6 +104,15 @@ edition = "2018"
 bevy = "0.1.2" # make sure this is the latest version
 ```
 
+### Make debug builds faster
+
+Debug Rust builds are notoriously slow, and this becomes an issue when making games. Luckily, you can set up rust so your dependendencies (including Bevy) are compiled with optimizations enabled, giving you faster runtime speeds without sacrificing debuggability or iteration speed with the ```--release``` flag. To enable this, append the following to your Cargo.toml file:
+
+```toml
+[profile.dev.packages."*"]
```

Comment:
```suggestion
[profile.dev.package."*"]
```

`cargo run` seems to throw a warning when `packages` is used instead of `package`

Reply:
Yup, thats a typo. I'll fix it when I open my computer.